frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx

```diff
@@ -0,0 +1,819 @@
+              >
+                <Typography variant="body2">
+                  {isPanelExtended ? t('Hide Table') : t('View Table')}
+                </Typography>
+              </Button>
+              <Button
+                className={classes.analysisButton}
+                onClick={clearAnalysis}
+              >
+                <Typography variant="body2">{t('Clear Analysis')}</Typography>
+              </Button>
+              <Button
+                className={classes.bottomButton}
+                onClick={() =>
+                  downloadCSVFromTableData(
+                    analysisResult,
+                    translatedColumns,
+                    selectedDate,
+                  )
+                }
+              >
+                <Typography variant="body2">{t('Download CSV')}</Typography>
+              </Button>
+            </div>
+          )}
+        {(!analysisResult ||
+          analysisResult instanceof ExposedPopulationResult) && (
+          <div className={classes.analysisButtonContainer}>
+            <Button
+              className={classes.bottomButton}
+              onClick={runAnalyser}
+              startIcon={<BarChartOutlined />}
+              disabled={
+                !!thresholdError || // if there is a threshold error
+                isAnalysisLoading || // or analysis is currently loading
+                !hazardLayerId || // or hazard layer hasn't been selected
+                (hazardDataType === GeometryType.Polygon
+                  ? !startDate || !endDate || !adminLevelLayerData
+                  : !selectedDate || !baselineLayerId) // or date hasn't been selected // or baseline layer hasn't been selected
+              }
+            >
+              <Typography variant="body2">{t('Run Analysis')}</Typography>
+            </Button>
+            {isAnalysisLoading ? <LinearProgress /> : null}
+          </div>
+        )}
+      </div>
+      {!isAnalysisLoading &&
+        analysisResult &&
+        (analysisResult instanceof BaselineLayerResult ||
+          analysisResult instanceof PolygonAnalysisResult) &&
+        isPanelExtended && (
+          <div className={classes.analysisTableContainer}>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                alignItems: 'center',
+              }}
+            >
+              <Typography className={classes.analysisTableTitle}>
+                {selectedHazardLayer?.title}
+              </Typography>
+              <IconButton
+                aria-label="close"
+                onClick={() => setIsPanelExtended(!isPanelExtended)}
+                className={classes.analysisTableCloseButton}
+              >
+                <CloseRounded />
+              </IconButton>
+            </div>
+            <AnalysisTable
+              tableData={analysisResult.tableData}
+              columns={translatedColumns}
+            />
```

Comment:
AnalysisTable and th remaining content of the analysis panel should be two elements of a flex-direction row. Only the table section should be scrollable